docs/07_how-to/20_prepare-test.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'How to prepare automated tests'
+sidebar_label: 'Prepare automated tests'
+id: ht-prepare-test
+keywords: [preparing tests, automated tests, testing]
+tags:
+    - preparing tests
+    - automated tests
+    - testing
+
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+Whichever component of your application you are testing, there is a simple set of tools that you can use. This page looks at those tools and the dependencies that you must declare before you can start automated testing.
+
+## Tools for testing
+
+Genesis uses the following tools for testing:
+
+- Cucumber BDD
```

Comment:
We mention uvu in our docs, do we not even use it?

Reply:
I see we mention it in web-unit test.. feels weird its not here, else we should mention here this is e2e tests (which we also have another section for below?!